src/redux/configureStore.js

```diff
@@ -0,0 +1,15 @@
+import { configureStore, combineReducers } from 'redux';
+import booksReducer from './books/books';
+import categoriesReducer from './categories/categories';
+
+const reducer = combineReducers({
+  booksReducer,
+  categoriesReducer,
+  // additional reducers could be added here
+});
+
+const store = configureStore(
+  reducer,
+);
+
+export default store;
```

Comment:
- [ ] [OPTIONAL] Kindly import `applyMiddleware` in your configureStore to be able to use some middleware as `redux-logger`